kubernetes/resource_kubernetes_service_test.go

```diff
@@ -366,6 +366,8 @@ func TestAccKubernetesService_nodePort(t *testing.T) {
 	var conf api.Service
 	name := acctest.RandomWithPrefix("tf-acc-test")
 	resourceName := "kubernetes_service.test"
+	appProtocolFirst := "ssh"
+	appProtocolSecond := "terraform.io/kubernetes"
```

Comment:
This is a recommended format for non-standard protocols(as per Kubernetes documentation)

Reply:
I was wondering if we could do any validation to this effect, but it seems the API server doesn't validate the field at all – it just accepts any free text.